src/docs/Components/TextInput/code.mdx

```diff
@@ -120,3 +120,36 @@ Wrap your input and use a 24px icon size :
   name="example"
 />
 ```
+
+**Required field**
```

Comment:
Can we use a heading, for it to appear in the table of content ? If it make sense.

Reply:
fixed